### PR TITLE
143 small fixes on example one page

### PR DESCRIPTION
**Pull request by daveatweaviate**
[issue](https://github.com/weaviate/weaviate-edu-team-tasks/issues/143)
[staging](https://650b5c2a22f8bb4be25a6f3d--tangerine-buttercream-20c32f.netlify.app/developers/academy/zero_to_mvp/hello_weaviate/examples_1)

Small fixes on the [example one page](https://weaviate.io/developers/academy/zero_to_mvp/hello_weaviate/examples_1)

### Type of change:
- [x] **Documentation** updates (non-breaking change to fix/update documentation)

### How Has This Been Tested?
- [x] **Github action** – automated build completed without errors
- [x] **Local build** - the site works as expected when running `yarn start`

### CODE REVIEW

File: developers/academy/zero_to_mvp/101_hello_weaviate/20_examples_1.mdx [hunk 1]
```diff
@@ -17,21 +17,30 @@ You will also compare vector search with keyword search to compare and contrast
...
 
-Imagine that you're running a quiz night, and want to get some questions around the category of "animals in movies". You could look for word matches - perhaps something like:
+Imagine that you're running a quiz night, and you want to get some questions about "animals in movies". In a traditional database you could look for word matches, perhaps something like:
 
 ```sql
 SELECT question, answer
 FROM jeopardy_questions
-WHERE (question LIKE '%animal%' OR question LIKE '%creature%' OR question LIKE '%beast%')
-AND (question LIKE '%movie%' OR question LIKE '%film%' OR question LIKE '%picture%' OR question LIKE '%cinema%')
+WHERE (
```
Comment (daveatweaviate) [inline developers/academy/zero_to_mvp/101_hello_weaviate/20_examples_1.mdx:27]: Reformatted to avoid horizontal scrolling

**Comment by daveatweaviate** [comment]
@databyjp  would you have a look at these updates when you have a moment? Thanks!

File: developers/academy/zero_to_mvp/101_hello_weaviate/20_examples_1.mdx [hunk 2]
```diff
@@ -181,7 +190,7 @@ Take a look at this response. Do you notice any additional information?
 
 </details>
 
-The difference is that the response now contains a `distance` value.
+The difference is that these responses contain a `distance` value.
```
Comment (databyjp) [inline developers/academy/zero_to_mvp/101_hello_weaviate/20_examples_1.mdx:193]: I think this should remain singular as there is just the one (represented in different ways). But I am being a real pedant. Happy to go with whatever.

**Comment by databyjp** [comment]
Hey @daveatweaviate I made one last comment - pls review & merge whenever :)

**Comment by daveatweaviate** [comment]
@databyjp  thanks for the feedback. Updated and merged as per slack

**Pull request merged**